Makefile

```diff
@@ -0,0 +1,36 @@
+EDX_EC_ARCHIVE_URL ?= https://github.com/openfun/ecommerce/archive/fun/ecommerce-ol.tar.gz
+
+default: help
+
+src/.keep:
+	mkdir -p src
+	touch src/.keep
+
+src/README.rst:
+	curl -Lo /tmp/ecommerce.tgz $(EDX_EC_ARCHIVE_URL)
+	tar xzf /tmp/ecommerce.tgz -C ./src --strip-components=1
+
+build: \
+  tree \
+  fetch-src
+build:  ## Build docker image
+	docker build -t edxec:latest .
+.PHONY: build
+
+clean:  ## Remove downloaded sources
+	rm -fr src
+.PHONY: clean
+
+fetch-src: src/README.rst
```

Comment:
We agree that this does not tell you if the sources are in the version actually targeted by your current $(EDX_EC_ARCHIVE_URL)

Reply:
Yes, we agree on this.

Reply:
This is why I've added the `clean` target. Do you have something more robust in mind?

Reply:
My point was that on our other Docker image projects, the src folder is used to store real git sources. The tar.gz release is downloaded directly in the dockerFile and never leaves the builder container... I was expecting the same scheme here but maybe it is not possible for a reason I can't see?

Reply:
That was a first attempt to debug the build. We should also download the sources in the Docker image.

Reply:
This is now implemented as expected, using the master branch of https://github.com/edx/ecommerce